silverback/runner.py

```diff
@@ -13,7 +13,7 @@
 from ape import chain
 from ape.logging import logger
 from ape.utils import ManagerAccessMixin
-from eth_utils import to_checksum_address
+from cchecksum import to_checksum_address
```

Comment:
wouldn't it make sense to re-export this via your package? (to maintain compatibility)